src/pip/_internal/commands/install.py

```diff
@@ -267,6 +267,11 @@ def run(self, options: Values, args: List[str]) -> int:
         if options.use_user_site and options.target_dir is not None:
             raise CommandError("Can not combine '--user' and '--target'")
 
+        # Prevent users from running pip install pip install, since this likely
+        # indicates a mistake with the command
+        if args[:2] == ["pip", "install"]:
+            raise CommandError("Likely incorrect command: pip install pip install ...")
```

Comment:
Perhaps you could further improve the message by suggesting the correct command? Perhaps something like
```suggestion
            raise CommandError(f"Likely incorrect command: pip install pip install ... Did you mean \"pip install {' '.join(args[2:-1])}\".")
```
But maybe that's an overkill. In any case, mostly just wanted to thank you on working on such a nice usability change! Cheers!